src/components/AppLike.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
```

Comment:
This component is only use for testing and since its location and naming is the same convention as components, we could consider this as an actual React component used by the application.
What about putting a test folder here? Or maybe put it directly in the related spec file?